User Flow Docs.sketchplugin/Contents/Sketch/export-for-user-flows.js

```diff
@@ -136,8 +136,13 @@ function hideScreenDescriptions(layers) {
 function _getSlices( artboard, sizes ) {
     if ( MSSliceMaker.slicesFromExportableLayer_sizes ) {
         return MSSliceMaker.slicesFromExportableLayer_sizes(artboard, sizes).objectEnumerator()
+    } else if ( MSSliceMaker.slicesFromExportableLayer_sizes_useIDForName ) {
+        return MSSliceMaker.slicesFromExportableLayer_sizes_useIDForName(artboard, sizes, false).objectEnumerator()
+    } else if ( MSSliceMaker.slicesFromExportableLayer ) {
+        return MSSliceMaker.slicesFromExportableLayer(artboard).objectEnumerator()
+    } else {
```

Comment:
Is it worth adding which version of Sketch each of these relates to? Likely not needed, but might be useful if you decide to drop support for some versions in the future.